JS-Algorithms/challenges.js

```diff
@@ -118,7 +173,20 @@ Because the 12th index in the Fibonacci sequence is 144, and 144 has three digit
 ***** */
 
 const fibIndex = (n) => {
-  // YOUR CODE HERE...
+  // offset index due to 1,1 being first two elements of fibonacci sequence
+  let index = 3 
```

Comment:
This may fail when `n = 1`, because the first element to have a length of 1, would be the first element of fibonacci.

Reply:
Tested with new code. Previously yes it returned index 3. 
![image](https://github.com/user-attachments/assets/db1d4c62-bd66-4f30-86e7-2fc0ca018428)